src/components/uicomposition/LayoutSubsection.astro

```diff
@@ -0,0 +1,53 @@
+---
+import type { ImageMetadata } from 'astro';
+
+import { Image } from 'astro:assets';
+interface Props {
+    alignment?: "column" | "row";
+    imageSrc?: string;
+    imageAlt?: string;
+    class?: string;
+}
+
+const { 
+    alignment = "column",
+    imageSrc,
+    imageAlt = '',
+    class: className,
+} = Astro.props;
+
+const images = import.meta.glob<{ default: ImageMetadata }>('/src/assets/images/*.{jpeg,jpg,png,gif}');
+if (imageSrc && !images[imageSrc]) throw new Error(`"${imageSrc}" does not exist in glob: "src/assets/images/*.{jpeg,jpg,png,gif}"`);
+---
+<section class:list={[
+    'relative my-4',
+    alignment == 'row' && '',
+    className,
+]}> 
+    { (alignment == 'row' && imageSrc) &&
+        <Image loading="lazy" class='w-1/2 shape-wrap lg:float-right my-8 mx-auto lg:ml-4 lg:mt-[8vh] lg:mb-4 lg:-mr-32 shadow-xl shadow-primary/20 rounded-xl overflow-hidden'
+            src={images[imageSrc]()} 
+            alt={imageAlt} />
+        <div class='text-justify'>
```

Comment:
I strongly advise not to use justify for text. It is proven that it is bad for usabilty and accessibility on the web.